test/runtime/src/lib.rs

```diff
@@ -244,6 +244,10 @@ impl pallet_balances::Config for Runtime {
 	type MaxLocks = ();
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
```

Comment:
I dont get the advantage of introducing the identifiers and then setting them all to `()` but it looks correct.

Reply:
Nothing is using this functionality yet; once pallets begin using it we'll need to increase the maximums and set the identifiers to sensible aggregates.